packages/remix-react/components.tsx

```diff
@@ -1068,9 +1070,13 @@ export function useSubmitImpl(key?: string): SubmitFunction {
           options as any
         ).submissionTrigger;
 
-        method = options.method || target.method;
-        action = options.action || target.action;
-        encType = options.encType || target.enctype;
+        method =
+          options.method || target.getAttribute("method") || defaultMethod;
+        action =
+          options.action || target.getAttribute("action") || defaultAction;
```

Comment:
I think this needs to be resolved now relative to the route hierarchy like Link (not the URL segments the way the browser does it)

Reply:
See other notes, I think this is fine!